devicetypes/fibargroup/fibaro-single-switch-2-zw5.src/fibaro-single-switch-2-zw5.groovy

```diff
@@ -125,6 +126,7 @@ def ping() {
 def installed(){
     log.debug "installed()"
     sendEvent(name: "checkInterval", value: 1920, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+    response(refresh())

```

Comment:
Seems odd to do this in installed. Shouldn't this be in configure?

Reply:
When I think initial state I think `installed`. Putting a refresh call in installed has been SOP for getting initial state for a while.